x-pack/plugins/maps/public/render_app.tsx

```diff
@@ -19,7 +19,7 @@ import {
   getToasts,
   getEmbeddableService,
   getDocLinks,
-  getSharedUXPluginContext,
```

Comment:
getSharedUXPluginContext is not longer used any where in maps. Can you remove from kibana_services.

Reply:
Done.